### PR TITLE
Updated color variables

### PR DESCRIPTION
**Pull request by himedlooff**
And some dependency updates.

### CODE REVIEW

**Comment by himedlooff** [comment]
I'll wait until this gets reviewed before updating other components.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,16 +2,21 @@ All notable changes to this project will be documented in this file.
...
-## 1.3.1 - 2014-11-02
+## 1.4.0 - 2015-01-13
 
-### Added
-- Nothing.
+### Changed
+- Replaces all CFPB color variables with non-CFPB colors. To add the CFPB theme
+  to your project you will need to the CFPB color palette and an Capital Framework
+  theme overrides file. Both files can be found in the generator-cf repo here:
+  <https://github.com/himedlooff/generator-cf/tree/dev/app/templates/src/static/css>
```
Comment (Scotchester) [inline CHANGELOG.md]: Update link to CFPB repo.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,16 +2,21 @@ All notable changes to this project will be documented in this file.
...
 
 
-## 1.3.1 - 2014-11-02
+## 1.4.0 - 2015-01-13
 
-### Added
-- Nothing.
+### Changed
+- Replaces all CFPB color variables with non-CFPB colors. To add the CFPB theme
+  to your project you will need to the CFPB color palette and an Capital Framework
```
Comment (Scotchester) [inline CHANGELOG.md]: Two grammatical errors in this sentence.

File: CHANGELOG.md (hunk 1 above)
Comment (himedlooff) [inline CHANGELOG.md]: Thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (himedlooff) [inline CHANGELOG.md]: Thanks!

File: Gruntfile.js [hunk 3]
```diff
@@ -20,6 +20,9 @@ module.exports = function(grunt) {
   require('jit-grunt')(grunt, {
     // Below line needed because task name does not match package name
     bower: 'grunt-bower-task'
+  })({
+    pluginsRoot: './node_modules/cf-grunt-config/node_modules',
+    customTasksDir: './node_modules/cf-grunt-config/tasks/options',
```
Comment (Scotchester) [inline Gruntfile.js]: What's this `customTasksDir` used for?

File: Gruntfile.js [hunk 4]
```diff
@@ -20,6 +20,9 @@ module.exports = function(grunt) {
...
   });
   // Sets the CWD back to the project root so that the tasks work as expected.
   grunt.file.setBase('../../');
```
Comment (Scotchester) [inline Gruntfile.js]: I think this needs to go away, too.

cf. [cf-expandables's Gruntfile](https://github.com/cfpb/cf-expandables/blob/gh-pages/Gruntfile.js)

File: bower.json [hunk 5]
```diff
@@ -22,8 +22,8 @@
   "license": "https://github.com/cfpb/cf-buttons/blob/gh-pages/TERMS.md",
   "main": "src/cf-buttons.less",
   "dependencies": {
-    "cf-core": "git://github.com/cfpb/cf-core.git",
-    "cf-icons": "git://github.com/cfpb/cf-icons.git"
+    "cf-core": "latest",
+    "cf-icons": "latest"
```
Comment (Scotchester) [inline bower.json:26]: These aren't registered Bower packages yet.

Comment (Scotchester) [inline bower.json:26]: Ugh, undo, undo. My first attempt at searching Bower's package repository failed.

Comment (Scotchester) [inline bower.json:26]: But, we don't want them to be `latest`, do we?

File: Gruntfile.js (hunk 3 above)
Comment (himedlooff) [inline Gruntfile.js]: After updating jit-grunt grunt would error out when running tasks. After adding these properties it worked.

File: bower.json (hunk 5 above)
Comment (himedlooff) [inline bower.json:26]: Yeah, that's how we've been targeting them. When you provide a git endpoint with no tag it pulls the default branch, which would be the latest and greatest.

File: Gruntfile.js (hunk 4 above)
Comment (himedlooff) [inline Gruntfile.js]: Thanks, this put me in the right direction, fixing now!

Comment (himedlooff) [inline Gruntfile.js]: Fixed.

**Comment by Scotchester** [comment]
:+1: thanks!

**Pull request merged**